README.md

```diff
@@ -271,6 +271,20 @@ ENV['ember-a11y-testing']: {
 }
 ```
 
+The color contrast check can cause undesired scrolling behavior.  To disable:
```

Comment:
I'd prefer to avoid having two examples right next to each other that are essentially the same thing. Can we modify the example above instead? Maybe we can word it as "For example, the `color-contrast` rule has some default scrolling functionality which you may want to disable like so:"